source/docs/07-tablature.xml

```diff
@@ -96,50 +89,102 @@
+                  <p>
+                     <figure>
+                        <head/>
+                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample005.txt" parse="text"/></egXML>
+                     </figure>
+                  </p>
+               </div>
+            </div>
+            <div xml:id="tablature-basics" type="div2">
+               <head>Encoding Tablatures</head>
+               <p>Unlike CMN, <gi scheme="MEI">note</gi> elements in tablature do not have an explicit pitch, but indicate playing instructions (i.e. where to place fingers on the fingerboard and when to play the strings). The <att>tab.course</att> attribute is used to capture which course is to be struck, and <att>tab.fret</att> specifies on which fret of the fretboard the player should stop the course. A value of <val>0</val> for <att>tab.fret</att> refers to an unstopped, open course; a value of <val>1</val> refers to the first fret; and so on. Course order is the same as that given in the <gi scheme="MEI">course</gi> elements.</p>
                <p>
                   <figure>
                      <head/>
-                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample354.txt" parse="text"/></egXML>
+                     <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/tablature/tablature-sample006.txt" parse="text"/></egXML>
                   </figure>
-               </p>
-               <p>Some instruments, like the 12-string guitar, have the four lowest strings tuned an octave above but are still written on a 6-line tablature staff. In this case, you may enumerate the open string pitches while maintaining 6 lines.</p>
+               </p>               
+               <p>A set of vertically-aligned symbols is indicated by <gi scheme="MEI">tabGrp</gi>, i.e., a chord whose notes are played at the same time, and have the same (minimum) notated duration. The duration of a <gi scheme="MEI">tabGrp</gi> may be indicated visually with a rhythm symbol, whose presence is encoded using the <gi scheme="MEI">tabDurSym</gi> element. In many cases, the rhythm sign is omitted where it would repeat the previous value. For this reason <gi scheme="MEI">tabDurSym</gi> is optional, and the durational value of the <gi scheme="MEI">tabGrp</gi> is encoded as <att>dur</att>.</p>
```

Comment:
```suggestion
               <p>A set of vertically-aligned symbols is indicated by <gi scheme="MEI">tabGrp</gi>, <abbr>i.e.</abbr>, a chord whose notes are played at the same time, and have the same (minimum) notated duration. The duration of a <gi scheme="MEI">tabGrp</gi> may be indicated visually with a rhythm symbol, whose presence is encoded using the <gi scheme="MEI">tabDurSym</gi> element. In many cases, the rhythm sign is omitted where it would repeat the previous value. For this reason <gi scheme="MEI">tabDurSym</gi> is optional, and the durational value of the <gi scheme="MEI">tabGrp</gi> is encoded as <att>dur</att>.</p>
```